src/main/java/ai/datahunters/md/server/analytics/AnalyticsService.java

```diff
@@ -0,0 +1,21 @@
+package ai.datahunters.md.server.analytics;
+
+import ai.datahunters.md.server.photos.PhotosRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.concurrent.ExecutionException;
+
+@Service
+public class AnalyticsService  {
+
+    private PhotosRepository repo;
+
+    @Autowired
+    public AnalyticsService(PhotosRepository repo) {
+        this.repo = repo;
+    }
+
+   public long photoCount() throws InterruptedException, ExecutionException {
+            return repo.count().get();
```

Comment:
you shouldn't block here. Please return Mono here and propagate it till the controller method body, please check the main photos controller for this